### PR TITLE
Remove astrometry_net_data refs

### PR DESCRIPTION
**Pull request by womullan**

### CODE REVIEW

File: config/hscConfig.py [hunk 1]
```diff
@@ -1,9 +1,7 @@
 """
-Configure to force using astrometry_net instead of directly
-loading Pan-STARRS1 reference catalogs
-"""
+THE ASTROMETRY_NET_DATA NO LONGER EXISTS IN PIPE_TASKS see
 
-from lsst.pipe.tasks.setConfigFromEups import setPhotocalConfigFromEups, setAstrometryConfigFromEups
+"""
```
Comment (jdswinbank) [inline config/hscConfig.py:5]: See... what?

File: config/hscConfig.py [hunk 2]
```diff
@@ -15,17 +13,6 @@
 config.processCcd.calibrate.astromRefObjLoader.retarget(LoadAstrometryNetObjectsTask)
 config.processCcd.calibrate.photoRefObjLoader.retarget(LoadAstrometryNetObjectsTask)
```
Comment (jdswinbank) [inline config/hscConfig.py]: Do we also need to take out the references to `LoadAstrometryNetObjectsTask`?

File: config/hscConfig.py (hunk 1 above)
Comment (womullan) [reply config/hscConfig.py:5]: ah yes I meant to add the ticket number .. which i did now

Comment (timj) [reply config/hscConfig.py:5]: I don't think we need a comment here to tell us the absence of something. The commit message would be sufficient and the merge commit would tell us the ticket number.

File: config/hscConfig.py (hunk 2 above)
Comment (womullan) [reply config/hscConfig.py]: ah perhaps... that was not necessarily obvious since it was reference to data which was being removed. Let me check

Comment (womullan) [reply config/hscConfig.py]: indeed tim says meas_extensions_astrometryNet was removed from the stack last year some time

**Comment by timj** [comment]
@wmwv you added this config file in DM-14272 -- I'm wondering if it still needs to exist at all. It's clear that nothing in lsst_ci is using it because if it was it would be complaining about meas_extension_astrometryNet not existing.

**Comment by wmwv** [comment]
HSC is not currently tested as part of the `lsst_ci` Python `tests` run by tests.

The script to do so is here in `lsst_ci/scripts/runHscQuickTest.sh` and this `config/hscConfig.py` supports that script.  It's quite possible that a decision could be made to include this very simple level of CI testing for HSC by adding the HSC script here in `lsst_ci` to the `tests`.  If so, then keeping the script and this config would be useful.

**Comment by womullan** [comment]
ok but that probably needs to be  another ticket - shall i leave the config file - not with standing TIm's point that it could not have worked recently.

**Pull request merged**